### PR TITLE
Add recipe for pytest-localftpserver plugin

### PR DESCRIPTION
**Pull request by leouieda**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by leouieda** [comment]
@s-weigand @oz123 I'm proposing this recipe to have the package available through the conda package manager. I'm keen to use this for one of my projects and it would help to have this. I included myself as a maintainer for the conda-forge package (basically just boils down to merging a PR when a release is made). Would either of you be interested in being a maintainer on the package as well? You can always be added/removed later if you want.

**Comment by leouieda** [comment]
@conda-forge/help-python this should be ready for review now

File: recipes/pytest-localftpserver/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - pip
+  run:
+    - python >=3.5
+    - pyftpdlib >=1.2.0
+    - pyopenssl
+    - pytest
+
+test:
+  imports:
+    - pytest_localftpserver
```
Comment (marcelotrevisani) [inline recipes/pytest-localftpserver/meta.yaml:29]: Can we try to add ``pip check`` here please?
```suggestion
    - pytest_localftpserver
  requires:
    - pip
  commands:
    - pip check
```

Comment (leouieda) [reply recipes/pytest-localftpserver/meta.yaml:29]: @marcelotrevisani  :+1: sure thing! (Valeu pela revisão rápida!)

**Comment by s-weigand** [comment]
I'm fine with being maintainer for this feedstock ^^

**Comment by marcelotrevisani** [comment]
If you want to, next time you can give a go to [grayskull](https://github.com/marcelotrevisani/grayskull) to generate recipes for projects on ``pypi`` :smile:

**Comment by leouieda** [comment]
@s-weigand done :slightly_smiling_face: 

> If you want to, next time you can give a go to grayskull to generate recipes for projects on pypi smile

@marcelotrevisani now that is really cool! Will definitely give it a shot next time

File: recipes/pytest-localftpserver/meta.yaml (hunk 1 above)
Comment (marcelotrevisani) [reply recipes/pytest-localftpserver/meta.yaml:29]: Tamo junto :smile:

**Pull request merged**